tests/testthat/test-german_map.R

```diff
@@ -0,0 +1,23 @@
+test_that("returns an object of the expected class", {
+  plot <- german_map(financial)
+  expect_s3_class(plot, "ggplot")
+})
+
+test_that("returns correct risk category values colors", {
+  data <- tibble(
+    postcode = c(53773L, 53774L, 53775L),
+    xctr_risk_category = c("high", "medium", "low"),
+    benchmark = rep("all", 3)
+  )
+  expected_colors <- list(rgb(1, 0, 0), rgb(1, 0.5, 0), rgb(0, 1, 0))
+
+  plot <- german_map(data)
```

Comment:
Note that the name `german_map()` has a structure `<subject>_<verb>()` that is inconsistent with the rest of the API (or so I think because I interpret "plot" as a verb in `plot_xxx()`) -- so you may consider `map_germany()` instead.